platform/node/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 
 ### ✨ New features
 - *...Add new stuff here...*
+* Compile Node targets without -std=c++11 option by @tdcosta100 in https://github.com/maplibre/maplibre-gl-native/pull/926
```

Comment:
Could you put this one under a new heading,
```
### 🐞 Bug fixes
*...Add new stuff here...*
```

Reply:
Yes, of course! Nice to have this section here.